File: Encryption/aes_encrypt.py
# AES Encryption Example
from cryptography.fernet import Fernet

# Generate a new AES key
key = Fernet.generate_key()

# Create a Fernet cipher instance with the key
cipher = Fernet(key)

# Encrypt a message
message = b"Check Algorithm, AES encryption!"
encrypted_message = cipher.encrypt(message)

# Decrypt the message
decrypted_message = cipher.decrypt(encrypted_message)

print("Original Message:", message)
print("Encrypted Message:", encrypted_message)
print("Decrypted Message:", decrypted_message)

File: Encryption/aes_encrypt.java
import javax.crypto.Cipher;
import javax.crypto.KeyGenerator;
import javax.crypto.SecretKey;
import javax.crypto.spec.SecretKeySpec;
import java.nio.charset.StandardCharsets;
import java.util.Base64;

public class aes_encrypt {
    public static void main(String[] args) throws Exception {
        // Generate a new AES key
        KeyGenerator keyGenerator = KeyGenerator.getInstance("AES");
        keyGenerator.init(128);
        SecretKey secretKey = keyGenerator.generateKey();

        // Create a cipher instance with the key
        Cipher cipher = Cipher.getInstance("AES");
        cipher.init(Cipher.ENCRYPT_MODE, secretKey);

        // Encrypt a message
        String message = "Hello, AES encryption!";
        byte[] encryptedMessage = cipher.doFinal(message.getBytes(StandardCharsets.UTF_8));

        // Decrypt the message
        cipher.init(Cipher.DECRYPT_MODE, secretKey);
        byte[] decryptedMessage = cipher.doFinal(encryptedMessage);

        System.out.println("Original Message: " + message);
        System.out.println("Encrypted Message: " + Base64.getEncoder().encodeToString(encryptedMessage));
        System.out.println("Decrypted Message: " + new String(decryptedMessage, StandardCharsets.UTF_8));
    }
}
